pyproject.toml

```diff
@@ -9,7 +9,6 @@ factory = "jupyter_packaging.npm_builder"
 ignore = ["tbump.toml", ".*", "*.yml", "package-lock.json", "bootstrap*", "conftest.py"]
 
 [tool.pytest.ini_options]
-addopts = "--doctest-modules"
```

Comment:
You can revert the doctest-releated changes, we fixed this in https://github.com/jupyter-server/jupyter_releaser/pull/246